driver/makedepend.ml

```diff
@@ -650,7 +657,9 @@ let run_main argv =
         \      from <file>";
       "-args0", Arg.Expand Arg.read_arg0,
         "<file> Read additional NUL separated command line arguments from \n\
-        \      <file>"
+        \      <file>";
+      "-strict", Arg.Set strict,
+      " if an input file does not exist, exit with code 2"
```

Comment:
Is it a good idea to specify the exact exit code? As far as I can see, `we exit with 2 whenever *any* error occurs, so it may be better to use a more general wording:
```suggestion
      " if an input file does not exist, exit with a nonzero exit code"
```

Reply:
sounds sensible, yes!